pom.xml

```diff
@@ -344,8 +341,28 @@
                     <findbugsXmlWithMessages>true</findbugsXmlWithMessages>
                     <xmlOutput>true</xmlOutput>
                     <excludeFilterFile>findbugs/findbugs-exclude.xml</excludeFilterFile>
+                    <effort>Max</effort>
```

Comment:
findbugs plugin versions and some of those configs are already defined as part of the common pom. You should probably inherit those and only add the settings that are not already in common (e.g. fail-on-error). This also ensure we don't have inconsistent plugin versions across projects.